scripts/configure.py

```diff
@@ -158,6 +158,8 @@
   },
   'lms' : {
     'apt': ['libcrypt-openssl-rsa-perl', 'libio-socket-ssl-perl'], # needed for ShairTunes2W support
+    'copy' : [{'from': 'bin/ARCH/squeezelite', 'to': 'streams/squeezelite'},
```

Comment:
`squeezelite` is already installed on these boxes; can you clarify what versions we are going from and to with this?

Reply:
I continue to not remember why this line was added
I'm going to simply remove it as the specific LMS / squeezelite install is managed elsewhere, testing just now and removing this line changes nothing